src/Pandell.PackageRunner/Program.cs

```diff
@@ -263,5 +266,12 @@ private static Parameters ParseArguments(IEnumerable<string> args)
 
             return parameters;
         }
+
+        /// <summary>
+        /// </summary>
+        private static byte[] HexStringToByteArray(string hex)
+        {
+            return Enumerable.Range(0, hex.Length/2).Select(x => Convert.ToByte(hex.Substring(x*2, 2), 16)).ToArray();
```

Comment:
Also, please use normal `for` loops where appropriate. `Enumerable.Range` will needlessly create objects where no memory pressure is neccessary. I understand that in this trivial app it doesn't matter at all, but a for loop the intention better IMO:

```c#
var result = new byte[hex.Length / 2];
for (var i = 0; i < result.Length; ++i)
{
    result[i] = Convert.ToByte(hex.Substring(i * 2, 2), 16);
}

return result;
```